src/main/java/net/aufdemrand/denizencore/DenizenCore.java

```diff
@@ -141,6 +144,7 @@ static void oncePerSecond() {
      * @param ms_elapsed how many MS have actually elapsed. (50 on a standard engine).
      */
     public static void tick(int ms_elapsed) {
+        currentTimeMillis = System.currentTimeMillis();

```

Comment:
Isn't this a bit heavy to call every tick? Why not just `currentTimeMillis += ms_elapsed`?